sources/packages/api/src/services/restriction/constants.ts

```diff
@@ -9,3 +9,24 @@ export const RESTRICTION_FEDERAL_MESSAGE =
  */
 export const RESTRICTION_PROVINCIAL_MESSAGE =
   "Your account has a hold on it. Please contact SABC to resolve.";
+/**
+ * Restriction code that are used in API.
+ * * The Restriction table will have more restriction
+ * * code, other than below mentioned codes.
+ */
+export enum RestrictionCode {
+  // When institution report withdrawal for a FT course application,
```

Comment:
I believe that it should be "When an institution report withdrawal for an FT course application".